### DISCUSSION TITLE
DIscord Invalid Link

### DISCUSSION THREAD

**Discussion by zarware** [vitodeploy/vito#252] [General]
Dunno if you are aware, but your Discord link within https://vitodeploy.com/introduction/contribution-guide.html is currently sending to an invalid invite ID.
Current link a to this message: https://discord.com/invite/dcUWA5DV

![Screenshot 2024-07-12 at 3 34 57 PM](https://github.com/user-attachments/assets/f00e2eb9-7f3d-4b48-bcd0-7fa3f75f9b59)

**Comment by saeedvaziry** [comment]
Hmm, will fix that. the correct link is on the top menu tho

https://discord.gg/uZeeHZZnm5
